client/app/bundles/KudosApp/components/TabBar.jsx

```diff
@@ -2,34 +2,37 @@ import React, { PropTypes } from 'react'
 import { observer } from 'mobx-react'
 import AppStore from '../stores/AppStore'
 
+const DropDownItem = ({ itemName, currentTab}) => {
+  const classNameForItem = 'nav-link' + (currentTab === itemName ? ' active' : '');
+
+  return <option className={`${classNameForItem} +  kudos-list__tab`} value={itemName}>
+    {itemName}
+  </option>
+};
+
 @observer
 export class TabBar extends React.Component {
+  static itemChanger(event) {
+    AppStore.kudosStore.setCurrentTab(event.target.value)
+  }
   render() {
-    const tabs = ['Recent', 'My Kudos', 'Awarded Kudos']
-
+    const drop_down = ['All Kudos', 'Kudos Received', 'Kudos Sent']
+    const gray = "#898989";
     return (
-      <ul className="kudos-list__nav">
-        {tabs.map(tab => (
-          <Tab
-            key={tab}
-            tabName={tab}
-            currentTab={AppStore.kudosStore.currentTab}
-            clickHandler={() => AppStore.kudosStore.setCurrentTab(tab)}
-          />
-        ))}
-      </ul>
-    )
-  }
-}
-
-const Tab = ({ tabName, currentTab, clickHandler }) => {
-  const classNameForTab = 'nav-link' + (currentTab == tabName ? ' active' : '')
-
-  return (
-    <li className="kudos-list__tab" onClick={clickHandler}>
-      <span className={classNameForTab}>{tabName}</span>
-    </li>
-  )
+    <div className="kudo_list-changer kudos-list">
+    <select onChange={TabBar.itemChanger} className="drop_down_menu">
+      {drop_down.map(drop_down => <DropDownItem key={drop_down}
+                                                itemName={drop_down}
+                                                currentTab={AppStore.kudosStore.currentTab}/>)}</select>
+      <svg width="100%" height="10">
+        <line x1="0" x2="100%" y1="5" y2="5"
+              stroke={gray}
+              strokeWidth="2.5"
+              strokeLinecap="square"
+              strokeDasharray=".5,5"
+              />
+      </svg>
+    </div>)};
```

Comment:
I tried locally and I think @trisarahsorus's prettier/husky thing isn't working at the moment.  As it is I think you need to clean up the style here a bit (for example, the return, div, and select all have the same indent).

Reply:
i got the restyling, sry bout that; I am actually not having any issues running it with the change to strings rather than objects containing the strings